web_src/js/features/migration.js

```diff
@@ -45,7 +45,7 @@ function checkItems(tokenAuth) {
   } else {
     enableItems = $user.val() !== '' || $pass.val() !== '';
   }
-  if (enableItems && $service.val() > 1) {
+  if (enableItems && $service.val() !== 'Git') {
```

Comment:
Should it be `'git'`?

Reply:
since I use .Title instead of .Name (witcho would return `""` when on PlainGitService) so `Git` is expected